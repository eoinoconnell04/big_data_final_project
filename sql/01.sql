-- find the number of users

SELECT COUNT(id_users) FROM users;
