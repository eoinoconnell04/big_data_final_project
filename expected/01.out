 count 
-------
     5
(1 row)
